backend/Messenger/Modules/Messenger.Conversations.GroupChats/Features/CreateGroupChat/CreateGroupChatCommand.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Messenger.Core.Requests.Abstractions;
+using Messenger.Core.Requests.Responses;
+
+namespace Messenger.Conversations.GroupChats.Features.CreateGroupChat;
+
+public record CreateGroupChatCommand(
+    Guid InitiatorId,
+    IEnumerable<Guid> InvitedMembers,
+    string Name
+) : ICommand<CreatedResponse<Guid>>;
```

Comment:
```suggestion
    string Name) : ICommand<CreatedResponse<Guid>>;
```
Последняя скобка остается на линии